### PR TITLE
fix: scroll to scroll alpha

### PR DESCRIPTION
**Pull request by omahs**
fix: scroll to scroll alpha

@Turupawn could you have a look at this fix?

### CODE REVIEW

File: src/multicall.ts [hunk 1]
```diff
@@ -596,13 +596,14 @@ export class Multicall {
...
       case Networks.lineaTestnet:
       case Networks.linea:
-      case Networks.scroll:
         return '0xcA11bde05977b3631167028862bE2a173976CA11';
       case Networks.etherlite:
         return '0x21681750D7ddCB8d1240eD47338dC984f94AF2aC';
       case Networks.zkSyncEra:
       case Networks.zkSyncEraTestnet:
         return '0xF9cda624FBC7e059355ce98a31693d299FACd963';
+      case Networks.scroll:
```
Comment (joshstevens19) [inline src/multicall.ts:605]: scrollAlpha

Comment (omahs) [reply src/multicall.ts]: Damn, sorry just changed this

**Comment by Turupawn** [comment]
@joshstevens19 @omahs looks good to me :+1:

**Comment by joshstevens19** [comment]
Merging will deploy Monday as away

**Pull request merged**